py/README.md

```diff
@@ -0,0 +1,374 @@
+# AllOfUs Python Client Library
+
+Python code in this library is used to access the AllOfUs Workbench API from
+the context of a Jupyter notebook. 
+
+## Installing the library
+
+The latest version of the library will be installed automatically on the Jupyter
+server your notebook code runs in, but if you need to upgrade to a version which 
+has not been released yet, you can do so by running in a notebook cell:
+
+```
+!pip install --user --upgrade 'https://github.com/all-of-us/pyclient/archive/<VERSION>.zip#egg=aou_workbench_client&subdirectory=py'
+```
+
+or for Python 2.7:
+
+```
+!pip2.7 install --user --upgrade 'https://github.com/all-of-us/pyclient/archive/<VERSION>.zip#egg=aou_workbench_client&subdirectory=py'
+```
+
+and then restarting your Python kernel.
+
+The modules you will use in your code are in the aou_workbench_client package.
+
+## Using cohorts in a notebook
```

Comment:
nit: Is it necessary to say this is being done in a Notebook? Are these instructions notebook-specific?